packages/reports/addon/templates/components/navi-request-preview.hbs

```diff
@@ -0,0 +1,60 @@
+{{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<div class="navi-request-preview" ...attributes>
+  <div class="navi-request-preview__table-container">
+    <table class="navi-request-preview__table">
+      <thead class="navi-request-preview__table-header">
+        {{#each columns as |col|}}
+          <th class={{concat "navi-request-preview__column-header" " " col.type}}>
```

Comment:
`col.type` might be too generic?  maybe `concat "navi-request-preview__column-header navi-request-preview__column-header-" col.type`?